core/commonMain/src/kotlinx/serialization/Serializers.kt

```diff
@@ -66,10 +82,10 @@ public fun SerializersModule.serializer(type: KType): KSerializer<Any?> =
  * Returns `null` if serializer cannot be created (provided [type] or its type argument is not serializable and is not registered in [this] module).
  */
 @OptIn(ExperimentalSerializationApi::class)
-public fun SerializersModule.serializerOrNull(type: KType): KSerializer<Any?>? {
-    return serializerByKTypeImpl(type, failOnMissingTypeArgSerializer = false)
-}
+public fun SerializersModule.serializerOrNull(type: KType): KSerializer<Any?>? =
+    serializerByKTypeImpl(type, failOnMissingTypeArgSerializer = false)
 
+// the `this` is empty if there is no need to search for a contextual serializer
```

Comment:
I'm not sure this comment explains anything, because empty module already means there's no context serializers to search (and you don't check anywhere if it's empty)

Reply:
A vestige from the old implementation